DRAFT.md

```diff
@@ -1589,6 +1583,28 @@ such types are copy constructible. We make `polymorphic` unconditionally copy
 and move constructible. The authors do not envisage that this could be relaxed
 in a future version of the C++ standard.
 
+While a copy constructor cannot be a template, we can conditionally constrain
+copy construction of `indirect`, in C++20 and later, by defining:
```

Comment:
While a copy constructor cannot be a template, in C++20 and later we can conditionally constrain
copy construction of `indirect` by defining: